grails-app/views/order/_orderItemFormDialog.gsp

```diff
@@ -198,7 +198,13 @@
             applyFocus("#product-suggest");
           },
           error: function (jqXHR, textStatus, errorThrown) {
-            $.notify("An error occurred", "errur");
+            if (jqXHR.responseText) {
+              let data = JSON.parse(jqXHR.responseText);
+              $.notify(data.errorMessage, "error");
+            }
+            else {
+              $.notify("An error occurred", "errur");
```

Comment:
Typo. This shouldn't matter since the default is error, but we should fix before merging.

Reply:
This notify was here before, I haven't noticed the typo. I'll fix that.